docs/backend/README.md

```diff
@@ -0,0 +1,53 @@
+# InstructLab Backend Working Group
+
+## Introduction
+
+The InstructLab project to date has produced a
+[CLI](https://github.com/instructlab/instructlab) that allows the execution of
+the InstructLab workflow on a variety of single-machine footprints.
+
+Some members of the InstructLab project also operate some backend services that
+produce regular full model builds using contributions to the
+[taxonomy](https://github.com/instructlab/taxonomy/). This runs in a larger
+clustered environment. This original backend is not built in a way that is
+easily reusable by others. It is a goal of the InstructLab project to produce an
+evolution of these backend capabilities that is more easily reusable by others.
+
+The InstructLab Backend Working Group represents a group of contributors
+interested in evolving the InstructLab backend capabilities. The group is open
+to all contributors interested in this work.
+
+## Group Membership
+
+There is no formal membership of this group. Those interested in contributing
+just need to show up and participate in the work of the group.
+
+## Primary Artifacts Produced by the Group
+
+The primary artifacts produced by the InstructLab Backend Working Group are
+architecture and design docs in the `docs/backend` directory of this repository.
+This is where the group will work out high-level planning for various components
+that should be created.
+
+Once individual components are created and implementation is in progress, code
```

Comment:
Where should the implementation (actual code) live? Is it TBD?

Reply:
TBD -- I would expect that to be laid out in some of the design docs created by the group and placed in this new directory.